### PR TITLE
feat: added product page and filters

### PR DESCRIPTION
**Pull request by hiteshverma27**

### CODE REVIEW

**Review by Creator-singh — APPROVED** [review]
looks good to me

File: Styles/product.css [hunk 1]
```diff
@@ -0,0 +1,26 @@
+main{
+    display: grid;
+    grid-template-columns: auto 1fr;
+}
+.filter{
+    width: 18VW;
```
Comment (viru2001) [inline Styles/product.css:6]: why capital words here for VW ? is there any specific reason for this

File: Styles/product.css [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    min-width: 250px;
+    top: 9vh;
+    overflow-y: auto;
+    height: 92vh;
+    border: 2px solid lightgray;
```
Comment (viru2001) [inline Styles/product.css:11]: you should not use color like this

File: Pages/product.html [hunk 3]
```diff
@@ -0,0 +1,851 @@
...
+    <meta name="description" content="Shop from the best collection of motorsport posters.">
+    <title>Products | LightsOut</title>
+</head>
+
+<body>
+    <nav class="w-100vw flex-space_between-center px-10 py-1 bg-white">
+        <a href="/index.html">
+            <h2>LightsOut</h2>
+        </a>
+        <input type="search" class="input w-40per" placeholder="Search LightsOut" />
```
Comment (viru2001) [inline Pages/product.html:22]: i think classname `w-40` is good than `w-40per`

**Review by viru2001 — COMMENTED** [review]
The code looks good. It's great that you tried to use many things from the component library. I gave few suggestions.

File: Styles/product.css (hunk 1 above)
Comment (hiteshverma27) [reply Styles/product.css:6]: No its a typo

File: Styles/product.css (hunk 2 above)
Comment (hiteshverma27) [reply Styles/product.css:11]: I will update it to var

File: Pages/product.html (hunk 3 above)
Comment (hiteshverma27) [reply Pages/product.html:22]: I have different units for utility classes for width for eg w-40per, w-30vw, w-40rem that's why

**Pull request merged**